### PR TITLE
Fixes dynamic growing cells in workflows

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
- The cells in the table of a workflow are dynamically growing
- The labels of the colors for the states are changed
- If multiple selection in motion-list is active, the button to change the view won't be seen

### CODE REVIEW

File: client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.ts [hunk 1]
```diff
@@ -404,6 +404,6 @@ export class WorkflowDetailComponent extends BaseViewComponent implements OnInit
      * @returns a string representing a color
      */
     public getStateCssColor(colorLabel: string): string {
```
Comment (FinnStutzenstein) [inline client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.ts]: This function is unnecessary and can be removed.

File: client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.ts [hunk 2]
```diff
@@ -93,7 +93,7 @@ export class WorkflowDetailComponent extends BaseViewComponent implements OnInit
     /**
      * Determine label colors. Where they should come from is currently now know
      */
-    public labelColors = ['default', 'primary', 'success', 'danger', 'warning'];
+    public labelColors: string[] = ['grey', 'red', 'green', 'lightblue', 'yellow'];
```
Comment (FinnStutzenstein) [inline client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.ts]: Are these the new colors? Is the order of this git-diff the same? I can write a migrtion for this, if you are sure not to touch the names again. So @GabrielInTheWorld are you sure that these names are good and won't be changed in the near future?

Comment (GabrielInTheWorld) [reply client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.ts]: As discussed with @emanuelschuetze these names are the new names for the states.
The names are the color itself, so the user see immediately, which color he selects and this color can be passed to the `mat-chip`s, as it is written.

File: client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.scss [hunk 3]
```diff
@@ -15,14 +15,6 @@ table {
         position: relative;
     }
 
-    // scaled up version of an inner table
-    .clickable-cell {
```
Comment (tsiegleauq) [inline client/src/app/site/motions/modules/motion-workflow/components/workflow-detail/workflow-detail.component.scss]: should not be removed

**Comment by FinnStutzenstein** [comment]
@GabrielInTheWorld Please test again.

**Pull request merged**